cdp-agentkit-core/cdp_agentkit_core/actions/wow/create_token.py

```diff
@@ -28,13 +29,16 @@ class WowCreateTokenInput(BaseModel):
     )
 
 
-def wow_create_token(wallet: Wallet, name: str, symbol: str) -> str:
+def wow_create_token(
+    wallet: Wallet, name: str, symbol: str, metadata_cid: Optional[str] = None
```

Comment:
please add the optional input argument to the argument schema class `WowCreateTokenInput`.

Also please update the `WOW_CREATE_TOKEN_PROMPT` to explain that the tool takes the token URI which stores the token metadata optionally and uses default metadata if not provided.

Reply:
```suggestion
    wallet: Wallet, name: str, symbol: str, token_uri: str | None = None
```